Sources/Sentry/include/SentryFramesTracker+Test.h

```diff
@@ -0,0 +1,12 @@
+#if SENTRY_HAS_UIKIT
+
+#    import "SentryFramesTracker.h"
+
+@interface SentryFramesTracker ()
+
+- (void)pause;
+- (void)unpause;
+
+@end
```

Comment:
`m`: You should be able to test this functionality by using the `TestNSNotificationCenterWrapper` and using the notifications `didBecomeActive` and `willResignActive` which call pause and unpause. So I think we can remove this or am I missing something?

https://github.com/getsentry/sentry-cocoa/blob/6b57184fb4aea6c8c6ad2765b3fe5f2a87f73710/Tests/SentryTests/Integrations/Performance/FramesTracking/SentryFramesTrackerTests.swift#L682-L689